### ISSUE TITLE
word2vec的python应用

### ISSUE THREAD

**Issue by XilongPei** [XilongPei/Openparts#71]
word2vec的python应用 https://blog.csdn.net/u012052268/article/details/78643260
https://www.oschina.net/p/word2vec/related

**Comment by XilongPei** [comment]
情感分析利器——Doc2vec https://blog.csdn.net/QFire/article/details/78653254

**Comment by XilongPei** [comment]
Word2vec的训练过程可以看做是通过神经网络机器学习算法来训练N-gram 语言模型，并在训练过程中求出word所对应的vector的方法。根据语言模型的不同，又可分为“CBOW”和“Skip-gram”两种模型。而根据两种降低训练复杂度的方法又可分为“Hierarchical Softmax”和“Negative Sampling”。两种模式和两种方法进行组合，所以实际上是有四种实现。https://blog.csdn.net/mpk_no1/article/details/72458003

**Comment by XilongPei** [comment]
中文文本标注工具Chinese-Annotator（内附多个开源文本标注工具） https://mp.weixin.qq.com/s?__biz=MzI4ODU5NjQ3OQ==&mid=2247484179&idx=1&sn=88bb57d70b9952465223887d354fa208&chksm=ec3d4b83db4ac29500b4391b2312fc69931c144258810e44389c82c14d6c29fcd8de81e5d343&scene=21#wechat_redirect
https://github.com/deepwel/Chinese-Annotator

**Comment by XilongPei** [comment]
常见文本相似度量方法总结 https://mp.weixin.qq.com/s?__biz=MzI4ODU5NjQ3OQ==&mid=2247484032&idx=1&sn=d9c07e7162f18a5fee341c54f3a44a5c&chksm=ec3d4a10db4ac3061f7f05b63b0f99012684b7940b0aa871b5e7ce84480fbd018bfe65b11dfc&scene=21#wechat_redirect

**Comment by XilongPei** [comment]
scikit-learn
Machine Learning in Python
Simple and efficient tools for data mining and data analysis
Accessible to everybody, and reusable in various contexts
Built on NumPy, SciPy, and matplotlib
Open source, commercially usable - BSD license http://scikit-learn.org/stable/#

**Comment by XilongPei** [comment]
GitHub 上有哪些有趣的关于 NLP 或者 DL 的项目？ https://www.zhihu.com/question/36853910

**Comment by XilongPei** [comment]
MXNet https://mxnet.incubator.apache.org/

已被 亚马逊 (Amazon) 选为 AWS 上的深度学习框架，支持动态图计算。MXNet 有许多中国开发者，因而有非常良好的中文文档支持。Gluon 接口使得 MXNet 像 Keras 一样简单易用。

**Comment by XilongPei** [comment]
[NLP] MXnet与TensorFlow的自然语言处理应用 https://www.jianshu.com/p/97494911d88f
TensorFlow提供了两个版本的word2vec实现，较简单的basic版和较复杂的advanced版。
